sdk/core/core-auth/tsconfig.json

```diff
@@ -1,7 +1,13 @@
 {
   "extends": "../../../tsconfig.package",
   "compilerOptions": {
```

Comment:
I haven't done much tsconfig changing beyond my basic app developer stuff. How can I learn more about these changes? Could you share what they're doing / why they are necessary?

Can be in a team meeting or 1:1 or not at all, just curious to learn!

Reply:
A lot of these changes are due to our use of [tshy](https://github.com/isaacs/tshy).  The documentation should help as to why we're doing certain things.